cfgov/v1/jinja2tags/__init__.py

```diff
@@ -0,0 +1,112 @@
+from six.moves import html_parser as HTMLParser
+
+from django.utils.module_loading import import_string
+
+from jinja2 import Markup, contextfunction
+from jinja2.ext import Extension
+
+from .datetimes import DatetimesExtension
+from .fragment_cache import FragmentCacheExtension
```

Comment:
I'm curious about the relative path imports here.  I think we're fairly inconsistent about this as-is, but had been under the impression we were moving towards explicit imports everywhere.  I have a slight preference for `from v1.jinja2tags.fragment_cache` even if it feels slightly redundant since we're inside that directory.

Reply:
I would definitely prefer absolute imports